README.md

```diff
@@ -6297,18 +6297,51 @@ with open('file.txt', 'w') as file:
 
 <details>
 <summary>How to print current working directory?</summary><br><b>
+
+    import os
+
+    current_working_directory = os.getcwd()
+    print(current_working_directory)
+
 </b></details>
 
 <details>
 <summary>Given the path <code>/dir1/dir2/file1</code> print the file name (file1)</summary><br><b>
-</b></details>
 
-<details>
-<summary>Given the path <code>/dir1/dir2/file1</code> print the name of the directory where the file resides (dir2)</summary><br><b>
+    import os
+
+    filename = os.path.basename('/dir1/dir2/file1')
+
+    # Another way
+    head, tail = os.path.split('/dir1/dir2/file1')
+    
+    # tail is everything after the final slash
+    filename = tail
```

Comment:
print(tail)

Reply:
Yes, sure. Just to confirm - I should `print(result)` sooner when possible, and not worry about the output besides the result like
```
print(tail)
>>> file1
```

Reply:
Yes. You create a new variable when you do something like: `filename = tail` and it is not necessary.
You can even do something like this: print(os.path.split('/dir1/dir2/file1')[1])

Reply:
Right. My only concern was readability of code. I suppose the audience is familiar with the language already, and just need confirmation answers.

Anyways, I made the changes and commited. Not sure if I should create a pull request, or click the resolve conversation. For now I'll wait for your response. Let me know if everything looks ok. Thank you.

Reply:
Looks good. Appreciate your effort. Merging